decidim-comments/app/cells/decidim/comments/comments_cell.rb

```diff
@@ -113,7 +113,7 @@ def single_comment?
       def single_comment
         return if options[:single_comment].blank?
 
-        @single_comment ||= model.comments.find_by(id: options[:single_comment])
+        @single_comment ||= SortedComments.for(model, id: options[:single_comment], order_by: order).first
```

Comment:
💬 Is the `SortedComments` class a bit overkill in this case? Couldn't we use `model.comments.not_hidden` to remove the moderated comments? 
Another option I can think would be to create a query base class that hides the moderated comments and use it here and as a base class for the `SortedComments` class. 

What do you think?

Reply:
I was thinking that since SortedComments is already used in that class, it would make sense to use it here as well. And also because the SortedComments class already has the functionality to filter a single comment. Why is it there if it's not supposed to be used?